dev/io.openliberty.org.apache.myfaces.4.0_fat/publish/servers/faces40_subscribeToEventTest/server.xml

```diff
@@ -0,0 +1,19 @@
+<!--
+    Copyright (c) 2023 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0
+ -->
+<server description="Server for testing Faces 4.0 subscribeToEvent">
+
+    <include location="../fatTestPorts.xml"/>
+
+    <featureManager>
+        <feature>faces-4.0</feature>
+        <feature>cdi-4.0</feature>
```

Comment:
No need to enable the CDI feature as it is automatically enabled as part of the faces-4.0 feature now.